### PR TITLE
fix: NDS Select props are broken

### PR DESCRIPTION
**Pull request by haideralsh**
## Description

Attempting to fix the NDSSelect Props

## Changes include

- [ ] breaking change: a change that is not backwards-compatible and/or changes current functionality
- [x] fix: a non-breaking change that solves an issue
- [ ] feature: a non-breaking change that adds functionality
- [ ] chore: contains no changes affecting the library, such as documentation or test updates

## Feature checklist

- [ ] Appropriate tests have been added 
- [ ] Documentation has been updated
- [ ] Accessibility has been considered

### CODE REVIEW

**Review by jherdman — COMMENTED** [review]
Could you cut a beta release for this so we can try it out?

File: package.json [hunk 1]
```diff
@@ -160,11 +162,10 @@
     "react-popper": "1.3.11",
     "react-popper-2": "npm:react-popper@2.2.4",
     "react-resize-detector": "^9.1.0",
-    "react-windowed-select": "^5.2.0",
-    "smoothscroll-polyfill": "^0.4.4",
     "react-select": "^5.8.0",
-    "styled-system": "^5.1.4",
-    "@types/styled-system": "5.1.22"
+    "react-window": "^1.8.10",
+    "smoothscroll-polyfill": "^0.4.4",
```
Comment (jherdman) [inline package.json]: Unrelated to this PR, but I think we can remove this: https://caniuse.com/css-scroll-behavior

File: src/Select/MenuList.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
+/*
+  Copied as is from: https://github.com/jacobworrel/react-windowed-select/blob/master/src/MenuList.tsx
```
Comment (jherdman) [inline src/Select/MenuList.tsx:2]: I believe we need to inline the license too https://github.com/jacobworrel/react-windowed-select/blob/master/LICENSE

File: src/Select/MenuList.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  const estimatedItemSize = Math.floor(totalHeight / itemCount);
+
+  const { innerRef, selectProps } = props;
+
+  const { classNamePrefix, isMulti } = selectProps || {};
+  const list = React.useRef<List>(null);
+
+  React.useEffect(() => {
+    setMeasuredHeights({});
+  }, [props.children]);
```
Comment (jherdman) [inline src/Select/MenuList.tsx:103]: This is a neat trick

**Review by jherdman — COMMENTED** [review]
Just a few initial notes

File: package.json (hunk 1 above)
Comment (haideralsh) [reply package.json]: I removed it.

File: src/Select/MenuList.tsx (hunk 2 above)
Comment (haideralsh) [reply src/Select/MenuList.tsx:2]: Done.

**Review by jherdman — APPROVED** [review]

**Pull request merged**